frontend/micro-ui/web/micro-ui-internals/packages/components-core/src/atoms/TextInput.js

```diff
@@ -28,7 +29,9 @@ const TextInput = (props) => {
   return (
     <React.Fragment>
       <div
-        className={`digit-text-input ${user_type === "employee" ? "" : "digit-text-input-width"} ${props?.className ? props?.className : ""}`}
+        className={`digit-text-input ${user_type === "employee" ? "" : "digit-text-input-width"} ${props?.className ? props?.className : ""} ${
+          variant === "error" ? "error" : ""
```

Comment:
class name keep as digit-field-error instead of error

Reply:
made the changes @jagankumar-egov